lgc/patch/PatchPreparePipelineAbi.cpp

```diff
@@ -297,33 +297,37 @@ void PatchPreparePipelineAbi::setCallingConv(ShaderStage shaderStage, CallingCon
 void PatchPreparePipelineAbi::setAbiEntryNames(Module &module) {
   bool hasTs = m_hasTcs || m_hasTes;
   bool isFetchless = m_pipelineState->getPalMetadata()->getVertexFetchCount() != 0;
+  bool isExportLess = m_pipelineState->isUnlinked() && !m_pipelineState->hasColorExportFormats() &&
+                      m_pipelineState->getPalMetadata()->getColorExportCount() != 0;
 
   for (auto &func : module) {
     if (!func.empty()) {
       auto callingConv = func.getCallingConv();
-      bool isFetchlessVs = false;
```

Comment:
As above: An exportless PS should have the same name as a normal PS, so none of this needs to happen.